andela/activo-web
# #159232837 Disable required fields when creating new asset category

**What does this PR do?**
It prefills and disables all required fields when creating a new asset category

**Description of Task to be completed?**
- add a disabled option to the required input fields and select box fields
- prepopulate required fields with the required data   

**How should this be manually tested?**
After cloning and running the app, navigate to the settings page and click on add new assets

- you should see the required input fields are disabled and prepopulated with the required data

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/story/show/159232837

**Screenshots (if appropriate)**
<img width="1440" alt="screen shot 2018-07-31 at 10 51 04 pm" src="https://user-images.githubusercontent.com/28359346/43489274-64b6a1e6-9514-11e8-9f3d-fcf69e111697.png">


**Questions:**
N/A
